DI-1299/README.md

```
@@ -0,0 +1,53 @@
+# Compare Somalier outputs in GRCh37 and GRCh38
+
+### Description
```

Markdownlint (0.37.0)

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)